wcfsetup/install/files/ts/WoltLabSuite/Core/Acp/Ui/User/Content/Remove/Clipboard.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Handles the user content remove clipboard action.
+ *
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2020 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/User/Content/Remove/Clipboard
+ * @since       5.4
+ */
+
+import AcpUiWorker from "../../../Worker";
+import * as Ajax from "../../../../../Ajax";
+import * as Language from "../../../../../Language";
+import UiDialog from "../../../../../Ui/Dialog";
+import { AjaxCallbackSetup } from "../../../../../Ajax/Data";
+import { DialogCallbackSetup } from "../../../../../Ui/Dialog/Data";
+import * as EventHandler from "../../../../../Event/Handler";
+
+interface AjaxResponse {
+  returnValues: {
+    template: string;
+  };
+}
+
+interface EventData {
+  data: {
+    actionName: string;
+    internalData: any[];
+    label: string;
+    parameters: {
+      objectIDs: number[];
+      url: string;
+    };
+  };
+  listItem: HTMLElement;
+}
+
+export class AcpUserContentRemoveClipboard {
+  public userIds: number[];
+  private readonly dialogId = "userContentRemoveClipboardPrepareDialog";
+
+  /**
+   * Initializes the content remove handler.
+   */
+  constructor() {
+    EventHandler.add("com.woltlab.wcf.clipboard", "com.woltlab.wcf.user", (data: EventData) => {
+      if (data.data.actionName === "com.woltlab.wcf.user.deleteUserContent") {
+        this.userIds = data.data.parameters.objectIDs;
```

Comment:
Avoid state in the class. Pass the userIDs as the second parameter of `Ajax.api()`.

Reply:
No I can't, because I need it during the execution of the worker.